### DISCUSSION TITLE
Purge css with certain prefix or from certain file.

### DISCUSSION THREAD

**Discussion by hasanm95** [FullHuman/purgecss#908] [Q&A]
Hi,
I am exploring purge CSS and I found there is a safelist for certain selectors that we don't want to remove. But, I want to remove only those CSS that start with certain prefixes and keep everything else as it was.

Thanks

**Comment by mupinnn** [comment]
Hi, did you already find any solution to this?

**Reply by hasanm95** [reply]
No man 😞
